diff --git a/vim_rs/Cargo.toml b/vim_rs/Cargo.toml
index 7993ccb..1c3964a 100644
--- a/vim_rs/Cargo.toml
+++ b/vim_rs/Cargo.toml
@@ -36,7 +36,7 @@ indexmap = "2.9"
 log = "0.4"
 miniserde = "0.1"
 quick-xml = { version = "0.39", optional = true }
-reqwest = { version = "0.12" }
+reqwest = { version = "0.12", default-features = false, features = ["rustls-tls-native-roots", "charset", "http2"] }
 thiserror = "2.0"
 tokio = { version = "1.44", features = ["rt-multi-thread", "macros", "sync"] }
 vim_macros = { version = "0.4.4", path = "../vim_macros" }
